### DISCUSSION TITLE
Some weapons have incorrect skill requirement tooltips

### DISCUSSION THREAD

**Discussion by HawkinsThe71st** [FakeFishGames/Barotrauma#14414] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The Autoshotgun, HMG and Machine Pistol have outdated skill requirement hint values.
Autoshotgun requires 75 weapons skill but tooltip states 60
HMG requires 90 weapons skill but tooltip states 75
Machine Pistol requires 50 weapons skill but tooltip states 60
![image](https://github.com/user-attachments/assets/4c12c960-3dc5-4db6-9185-3234053f2afb)

### Reproduction steps

securityofficer_talent_items.xml and weapons.xml

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/b29bce986a38acc03d746639920585fe5ed057a8 (commit in our private repo).
